doc/threat_model/threat_model.md

```diff
@@ -216,6 +224,31 @@ The FPGA design as well as the firmware has been audited, and
 hardening of these has been performed to some degree. For more
 information, see the [Release Notes](/doc/release_notes.md)
 
+#### Mitigations
+
+- To protect the UDS the hardware design allows only one read per word
+  of the UDS per power-cycle.
+
+- USB port attacks - boot protocol:
+
+  - The firmware has a more strict protocol state machine and exits out
```

Comment:
What happens in the eternal loop? Should we instead trigger a trap similar to the read outside of the RAM memory?

Reply:
Answering here for traceability: this is how it used to be, and are the previous release. Under unreleased we have the switch from loop to hardware trap.